.gitattributes

```diff
@@ -18,6 +18,7 @@ bld.bat text eol=crlf
 .gitignore linguist-generated=true
 .travis.yml linguist-generated=true
 .scripts/* linguist-generated=true
+.woodpecker.yml linguist-generated=true
```

Comment:
Assuming this is due to a Linguist update, all good ...

Reply:
Once again, automated files. Let's leave them be.